games-board/crafty/crafty-24.1-r2.ebuild

```diff
@@ -1,13 +1,13 @@
 # Copyright 1999-2023 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=6
+EAPI=8
+
 inherit flag-o-matic toolchain-funcs
 
 DESCRIPTION="Bob Hyatt's strong chess engine"
-HOMEPAGE="http://www.craftychess.com/"
-SRC_URI="http://www.craftychess.com/${P}.zip
-	http://www.cis.uab.edu/hyatt/crafty/source/${P}.zip
+HOMEPAGE="https://web.archive.org/web/20231106192545/http://craftychess.com/"
+SRC_URI="https://web.archive.org/web/20210304102649/http://www.craftychess.com/downloads/source/${P}.zip
 	ftp://ftp.cis.uab.edu/pub/hyatt/documentation/${PN}.doc.ascii"
```

Comment:
Turns out this is dead too. `ftp.cis.uab.edu` doesn't even resolve.

Reply:
Any suggestion for replacement? The file is on Gentoo mirrors, so it's not really a problem. (We could use `mirror://gentoo/` but that won't change much.)

Reply:
It see the file on https://distfiles.macports.org/crafty/24.1/ but that's not better than our own mirrors.